eng/common/pipelines/templates/steps/sparse-checkout.yml

```diff
@@ -38,8 +38,8 @@ steps:
             } else {
               Write-Host "Repository $($repository.Name) is being initialized."
 
-              Write-Host "git clone --no-checkout --filter=tree:0 git://github.com/$($repository.Name) ."
-              git clone --no-checkout --filter=tree:0 git://github.com/$($repository.Name) .
+              Write-Host "git clone --no-checkout --filter=tree:0 https://github.com/$($repository.Name) ."
```

Comment:
Same question re: ACR anon access.

Reply:
Yeah, these changes are needed to keeping the tooling happy. CI builds now have additional checks and I brought the minimum changes in to pass the CI checks (cspell and sparse-checkout fixes).